pages/resources/insights.mdx

```diff
@@ -0,0 +1,185 @@
+# Insights
```

Comment:
the api routes are all under /analytics so I'm wonder if we rename this file to analytics.mdx instead? not dead set on this tho, pls bikeshed